src/main/kotlin/stringcalculator/InputNumber.kt

```diff
@@ -0,0 +1,24 @@
+package stringcalculator
+
+data class InputNumber(
+    val value: Int
```

Comment:
```suggestion
    private val value: Int
```
여기도 `private` 로 선언해주면 될 것 같습니다 :)